appHooks/apphub/pre.js

```diff
@@ -18,11 +18,16 @@ class AppHub extends AppHook {
   async test () {
     const apphub = await this.file('.apphub')
 
-    if (apphub == null) return
+    if (apphub == null) {
+      this.error('exist')
+      return
+    }
 
     let data = {}
     try {
-      data = JSON.parse((new Buffer(apphub, 'base64')).toString())
+      const buffer = new Buffer(apphub, 'base64').toString()
+      if (buffer.indexOf("{") == -1) return
```

Comment:
For some reason the blank apphub file I have in my testing repo ended up with two blank lines. I guess we could do something like strip all blank lines and spaces and see if the result is a blank string instead.